.eslintrc.json

```diff
@@ -1,26 +1,21 @@
 {
-    "env": {
-        "browser": true,
-        "es6": true,
-        "node": true
+  "env": {
+    "browser": true,
+    "es6": true,
+    "node": true
+  },
+  "extends": ["airbnb", "prettier"],
```

Comment:
The only change I made in this file was adding `"prettier"` to this `extends` array.
The rest of the changes are just formatting changes from running Prettier over the codebase.